litellm/pytest_plugin.py

```diff
@@ -0,0 +1,7 @@
+import pytest
+import litellm
+
+
+@pytest.fixture(autouse=True)
```

Comment:
please dont place this on the root, you can put it in `tests/` dir

Reply:
The point of putting this fixture inside the `litellm/` folder, is that the fixture is shipped to litellm users. If I put this in the `tests/` dir, it will not get packaged with litellm and is not able to be reused by users calling async litellm in their tests.

Or do you mean that this fixture is `autouse=True` by default?

Reply:
ok makes sense